src/js/providers/html5.js

```diff
@@ -482,9 +482,6 @@ define([
                 // Playback rate is broken on Android HLS
                 _this.supportsPlaybackRate = false;
             }
-            if (source.preload && source.preload !== _videotag.getAttribute('preload')) {
-                _setAttribute('preload', source.preload);
-            }
```

Comment:
Are you made sure that the `preload` attribute is being set on the video tag before `src` is set? If not, then the browser's default behavior will kick in before `provider. preload` is called. This would be bad in the case where preload should be set to none or metadata but the browser's default is to use auto.